DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: epiworldR
 Type: Package
 Title: Fast Agent-Based Epi Models
-Version: 0.5-0
+Version: 0.6-0
```

Comment:
Is that the new version in C++?

Reply:
It will be after the C++ PR is merged

Reply:
Alright. Can you create an issue (if there's not one yet) about how to deal with versions in epiworld? I am not comfortable bumping the versions as we are doing it so far.

Reply:
#55 is for this problem